packages/ui/src/components/ResourceBlock/ResourceBlock.tsx

```diff
@@ -0,0 +1,194 @@
+    "data-h2-border-bottom": "base(quaternary.darkest)",
+  },
+  quinary: {
+    "data-h2-background-color": "base(quinary.lightest)",
+    "data-h2-border-bottom": "base(quinary.darkest)",
+  },
+};
+
+const wrapperStyleMap: Record<CardColor, Record<string, string>> = {
+  primary: {
+    "data-h2-color": "base(primary.darkest)",
+  },
+  secondary: {
+    "data-h2-color": "base(secondary.darkest)",
+  },
+  tertiary: {
+    "data-h2-color": "base(tertiary.darkest)",
+  },
+  quaternary: {
+    "data-h2-color": "base(quaternary.darkest)",
+  },
+  quinary: {
+    "data-h2-color": "base(quinary.darkest)",
+  },
+};
+
+interface RootProps {
+  title: ReactNode;
+  headingColor?: CardColor;
+  headingAs?: HeadingLevel;
+  children: ReactElement<ItemProps> | Array<ReactElement<ItemProps>>; // Restricts children to only expected items;
+}
+
+const Root = ({
+  title,
+  headingColor = "primary",
+  headingAs = "h3",
+  children,
+}: RootProps) => {
+  // prepare heading text element
+  const HeadingTextElement = headingAs;
+  const headingTextStyles = { ...headingComponentStyles["h4"] };
+  delete headingTextStyles["data-h2-margin"];
+  return (
+    <div
+      data-h2-shadow="base(larger)"
+      data-h2-border-radius="base(rounded)"
+      data-h2-background-color="base(foreground)"
+    >
+      {/* heading bar */}
+      <div
+        {...headingBarStyleMap[headingColor]}
+        data-h2-border-radius="base(rounded rounded 0 0)"
+        data-h2-border-bottom-width="base(1px)"
+        data-h2-border-bottom-style="base(solid)"
+        data-h2-padding="base(x1 x1.5 x1 x1.5)"
+        data-h2-text-align="base(center)"
+      >
+        {/* wrapper */}
+        <div {...wrapperStyleMap[headingColor]}>
+          <HeadingTextElement {...headingTextStyles}>
+            {title}
+          </HeadingTextElement>
+        </div>
+      </div>
+      {/* content */}
+      <div data-h2-display="base(flex)" data-h2-flex-direction="base(column)">
+        {children}
+      </div>
+    </div>
+  );
+};
+
+interface ItemProps {
+  link: ReactElement<LinkProps>;
+  description: string;
+  state?: "incomplete" | "complete";
+}
+
+const getStateIcon = (state: ItemProps["state"]): ReactNode | null => {
+  const commonStyles: HydrogenAttributes = {
+    "data-h2-width": "base(20px)",
+    "data-h2-height": "base(20px)",
+    "data-h2-position": "base(absolute)",
+    "data-h2-location": "base(x0.75, x0.75, auto, auto)",
+  };
+
+  if (state === "incomplete") {
+    return (
+      <ExclamationCircleIcon data-h2-color="base(error)" {...commonStyles} />
+    );
+  }
+  if (state === "complete") {
+    return <CheckCircleIcon data-h2-color="base(success)" {...commonStyles} />;
+  }
+  return null;
+};
+
+const Item = ({ link: rawLink, description, state }: ItemProps) => {
+  // prepare link
+  const link = rawLink
+    ? cloneElement(rawLink, {
+        color: "black",
+        // yuck, style exception 😞
+        "data-h2-font-weight": "base(bold)",
```

Comment:
I'm not sure what to do about this.  The design has a bolded link but our standard links don't look like that.  Style exceptions tend to bite us when it comes time to make site-wide style changes.

Reply:
[cloneElement](https://react.dev/reference/react/cloneElement) is an old react feature. For this, you can just grab the `link: { href: string; label: string; }` as a prop. Then just use the `<Link />` component we have directly. Then remove the link variable.

```tsx
 <Link
      href={link.href}
      data-h2-font-weight="base(bold)"
      data-h2-position="base:selectors[::after](absolute)"
      data-h2-content="base:selectors[::after](' ')"
      data-h2-inset="base:selectors[::after](0)"
      data-h2-justify-self="base(end)"
      color="black"
    >
      {link.label}
 </Link>
 ```

Reply:
Interesting, I didn't know that function was not recommended.  I like the new solution better anyway.
[refactor link props](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11200/commits/c7d7cdb5a3b7f3ee66175b0b02d018721671b3f0)